benchmarks/benchmark-micronaut-data-jdbc/build.gradle

```diff
@@ -10,4 +10,5 @@ dependencies {
     implementation "io.micronaut.sql:micronaut-jdbc-hikari"
     runtimeOnly "com.h2database:h2"
     runtimeOnly mn.snakeyaml
+    runtimeOnly mnSerde.micronaut.serde.support
```

Comment:
Now that we changed from codecs to serde and have compileOnly dependency in data-jdbc, all projects using jdbc might need to import serde? I had dependency as implementation, but that caused issue for test resources.